specs/stork-daemonset.yaml

```diff
@@ -83,7 +83,7 @@ spec:
       port: 8099
       targetPort: 8099
 ---
-apiVersion: extensions/v1beta1
```

Comment:
This file is missing a spec selector and does not apply:

```
error: error validating "stork-daemonset.yml": error validating data: ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec
```

Reply:
Fixed.